Cargo.toml

```diff
@@ -165,6 +162,11 @@ codegen-units = 1
 inherits = "production"
 strip = true
 
+[profile.profiling]
+inherits = "production"
```

Comment:
Maybe we should inherit from `release` instead and set `debug-assertions = false`?
Reasoning:
- `codegen-units = 1` which slows compile-time down a lot
- There are special `cfg`s for the production profile in the code, so it would be wierd to inherit from production, but not enable the `cfg`s (I think its mostly including the resource folder into the binary).

Reply:
good point, having a profile inherit from production without wanting or getting the production resource logic would probably be confusing. i’ll fix that.

single codegen unit does slow build times, yes, but the splitting may have an [unpredictable impact on runtime perf](https://internals.rust-lang.org/t/lets-talk-about-parallel-codegen/2759/6) from build to build, unless rust still splits codegen units by module [like it did in 2015](https://internals.rust-lang.org/t/lets-talk-about-parallel-codegen/2759). i’m looking into this as we speak.

Reply:
done in 2b108c007881d67ecff51bd374f4a8ef98e7746a; codegen units addressed in description